src/main/java/hudson/plugins/logparser/LogParserParser.java

```diff
@@ -40,7 +41,7 @@
 
     public LogParserParser(final FilePath parsingRulesFile,
             final boolean preformattedHtml, final VirtualChannel channel)
-            throws IOException {
+            throws IOException, InterruptedException {

```

Comment:
Actually, not sure why it was added given that the constructor code didn't change.